src/modules/shared.mjs

```diff
@@ -105,92 +76,227 @@ export function addFeedbackLink() {
   document.body.appendChild(aside);
 }
 
-export function updateLogo() {
-  const logo = document.getElementsByTagName("img")[0];
-  logo.src = "https://beta.nj.gov/files/dol_logo.png";
-  logo.style.removeProperty("width");
-  logo.style.removeProperty("height");
-  logo.style.marginLeft = isDesktop() ? "107px" : "18px";
-  logo.alt = "Official logo for the New Jersey Department of Labor";
-  logo.parentElement.parentElement.removeAttribute("height");
+// Used for tagged template strings
+export function html(strings, ...values) {
+  return String.raw({ raw: strings }, ...values);
 }
 
-// Used for tagged template strings
-const html = (strings, ...values) => String.raw({ raw: strings }, ...values);
+export function js(strings, ...values) {
+  return String.raw({ raw: strings }, ...values);
+}
+
+export function css(strings, ...values) {
+  return String.raw({ raw: strings }, ...values);
+}
```

Comment:
Dust: are these just separate for readability? Just wondering why not just have one function for this?

Reply:
This is because when you put "js" or "css" before a tagged template, your IDE will automatically format and highlight the string as if it were actually CSS/JS/HTML. So this is just a function that will prevent an error from showing when you do `const text = html`<div></div>``

Reply:
Ty for explanation! I didn't realize that!